doc/governance.rst

```diff
@@ -139,6 +141,71 @@ Team leads are selected by the executive council, generally based on
 the recommendation by team members and outgoing lead.  Periodic
 rotation of lead responsibility among team members is encouraged.
 
+Current Component Leads
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Component
+     - Description
+   * - Adam King
+     - Cephadm
+     - Ceph's native orchestrator (install, upgrade...)
+   * - Venky Shankar
+     - CephFS
+     - Shared filesystem
+   * - Matan Breizman
+     - Crimson
+     - Next generation implementation of the OSD
+   * - Afreen Misbah
+     - Dashboard
+     - Ceph's graphical user interface
+   * - Radoslaw Zarzynski
+     - RADOS
+     - Ceph's low level storage foundation
+   * - Ilya Dryomov
+     - RBD
+     - Block storage for containers and VMs
+   * - Adam Emerson, Eric Ivancich
+     - RGW
+     - S3/Swift compatible object storage
+   * - Yingxin Cheng
+     - Seastore
+     - Next generation storage underlying the OSD
+
+
+General Leads
+=============
+
+These are general areas rather than specific subsystems within Ceph.
+Each is unique.
+
+Documentation
```

Comment:
@zdover23 can you fill in this part? I just added some basics

Reply:
@jdurgin, I've added a small list of things that I do.

Reply:
thanks zac, looks good!